common/src/main/java/com/wynntils/models/items/ItemModel.java

```diff
@@ -34,18 +34,8 @@
 import com.wynntils.models.items.annotators.game.TomeAnnotator;
 import com.wynntils.models.items.annotators.game.TrinketAnnotator;
 import com.wynntils.models.items.annotators.game.UnknownGearAnnotator;
-import com.wynntils.models.items.annotators.gui.AbilityTreeAnnotator;
-import com.wynntils.models.items.annotators.gui.ActivityAnnotator;
-import com.wynntils.models.items.annotators.gui.ArchetypeAbilitiesAnnotator;
-import com.wynntils.models.items.annotators.gui.CosmeticTierAnnotator;
-import com.wynntils.models.items.annotators.gui.DailyRewardMultiplierAnnotator;
-import com.wynntils.models.items.annotators.gui.IngredientPouchAnnotator;
-import com.wynntils.models.items.annotators.gui.SeaskipperDestinationAnnotator;
-import com.wynntils.models.items.annotators.gui.ServerAnnotator;
-import com.wynntils.models.items.annotators.gui.SkillCrystalAnnotator;
-import com.wynntils.models.items.annotators.gui.SkillPointAnnotator;
-import com.wynntils.models.items.annotators.gui.SoulPointAnnotator;
-import com.wynntils.models.items.annotators.gui.TerritoryAnnotator;
+import com.wynntils.models.items.annotators.gui.*;
```

Comment:
Please configure your Artemis IDEA project settings to not use wildcard imports (and static imports).